src/analyze/analyze-verify.c

```diff
@@ -124,10 +124,13 @@ static int verify_socket(Unit *u) {
         return 0;
 }
 
-static int verify_executable(Unit *u, ExecCommand *exec) {
+int verify_executable(Unit *u, ExecCommand *exec) {
```

Comment:
While at it, please make this take `const ExecCommand*`.

Reply:
I made this change, thanks for the suggestion.